support/setup_local_env.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# set -euo pipefail
+
+export HAB_LICENSE="accept-no-persist"
+sudo -E hab pkg install core/bzip2
+sudo -E hab pkg install core/libarchive
+sudo -E hab pkg install core/libsodium
+sudo -E hab pkg install core/openssl
+sudo -E hab pkg install core/xz
+sudo -E hab pkg install core/zlib
+sudo -E hab pkg install core/zeromq
+sudo -E hab pkg install core/protobuf --binlink
+sudo -E hab pkg install core/rust --binlink
+
+# for the libarchive crate
+export LIBARCHIVE_LIB_DIR
+LIBARCHIVE_LIB_DIR="$(hab pkg path core/libarchive)/lib"
+export LIBARCHIVE_INCLUDE_DIR
+LIBARCHIVE_INCLUDE_DIR="$(hab pkg path core/libarchive)/include"
```

Comment:
OPTIONAL: this could be a little less repetitive:
```bash
LIBARCHIVE_PKG_PATH=$(hab pkg path core/libarchive)
export LIBARCHIVE_LIB_DIR="$LIBARCHIVE_PKG_PATH/lib"
export LIBARCHIVE_INCLUDE_DIR="$LIBARCHIVE_PKG_PATH/include"
```
and later where its referenced in `LD_LIBRARY_PATH`.

Similarly for other `hab pkg path` calls.